src/liballoc/collections/linked_list.rs

```diff
@@ -610,8 +670,10 @@ impl<T> LinkedList<T> {
     /// assert_eq!(dl.front().unwrap(), &1);
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
-    pub fn push_front(&mut self, elt: T) {
-        self.push_front_node(box Node::new(elt));
+    pub fn push_front_alloc(&mut self, elt: T) -> Result<(), A::Err> {
```

Comment:
change to a stable method? insta-stabilising a method?

Reply:
:+1:  Right the docs and annotations should stay with the `push_front`, which is now in a different impl.